Gemfile

```diff
@@ -1,4 +1,35 @@
+#
+#  Gemfile
+#  BuildTools
+#
+#  Created by Christian Noon on 4/30/19.
+#  Copyright © 2018 Nike. All rights reserved.
+#
+
 source "https://rubygems.org"
 
-gem "cocoapods", "~> 1.6.0.beta.1"
-gem "xcpretty"
+gem "aws-sdk-s3", "~> 1.36.0"
+gem "cocoapods", "~> 1.7.0.beta.3"
+gem "danger", "~> 6.0.5"
+gem "danger-xcov", "~> 0.4.1"
+gem "fastlane", "~> 2.120.0"
+gem "jenkins_api_client", "~> 1.5.3"
+gem "json", "~> 2.2.0"
+gem "nokogiri", "~> 1.10.2"
+gem "plist", "~> 3.5.0"
+gem "pry", "~> 0.12.2"
+gem "psych", "~> 3.1.0"
+gem "xcov", "~> 1.5.0"
+gem "xcpretty-json-formatter", "~> 0.1.0"
+
+group :development do
+  gem "rspec", "~> 3.8.0", require: false
+  gem "rspec_junit_formatter", "~> 0.4.1", require: false
+  gem "rubocop", "~> 0.67.2", require: false
+  gem "rubocop-performance", "~> 1.1.0", require: false
+  gem "simplecov", "~> 0.16.1", require: false
+  gem "simplecov-json", "~> 0.2.0", require: false
+end
+
+plugins_path = File.join(File.expand_path(__dir__), "Fastlane", "Pluginfile")
+eval_gemfile(plugins_path) if File.exist?(plugins_path)
```

Comment:
Needs copyright header update and should only be `cocoapods` and `xcpretty`. We don't need the others for this project.

Reply:
Modified copyright header to proper year and trimmed the gems down as suggested and re-installed to update the lockfile.